python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
   - name: g_bdFluxDErr
     "@id": "#Object.g_bdFluxDErr"
     datatype: double
     description: Flux uncertainty from the exponential fit. Measured on g-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 2137
   - name: g_bdReB
     "@id": "#Object.g_bdReB"
     datatype: double
     description: Half-light ellipse of the de Vaucouleurs fit. Measured on g-band.
     fits:tunit: pixel**2
+    tap:principal: 0
+    tap:column_index: 2138
   - name: g_bdReD
     "@id": "#Object.g_bdReD"
     datatype: double
     description: Half-light ellipse of the exponential fit. Measured on g-band.
     fits:tunit: pixel**2
+    tap:principal: 0
+    tap:column_index: 2139
   - name: g_blendedness
     "@id": "#Object.g_blendedness"
     datatype: double
     description: Measure of how much the flux is affected by neighbors, (1 - child_flux/parent_flux).  Operates on the absolute value of the pixels to try to obtain a de-noised value.  See section 4.9.11 of Bosch et al. 2018, PASJ, 70, S5 for details. Measured on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2140
   - name: g_blendedness_flag
     "@id": "#Object.g_blendedness_flag"
     datatype: boolean
     description: General Failure Flag. Measured on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2141
   - name: g_cModelFlux
     "@id": "#Object.g_cModelFlux"
     datatype: double
     description: Flux from the final cmodel fit. Forced on g-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 2142
   - name: g_cModelFluxErr
     "@id": "#Object.g_cModelFluxErr"
     datatype: double
     description: Flux uncertainty from the final cmodel fit. Forced on g-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 2143
   - name: g_cModelFlux_inner
     "@id": "#Object.g_cModelFlux_inner"
     datatype: double
     description: Flux within the fit region, with no extrapolation. Forced on g-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 2144
   - name: g_cModel_flag
     "@id": "#Object.g_cModel_flag"
     datatype: boolean
     description: Flag set if the final cmodel fit (or any previous fit) failed. Forced on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2145
   - name: g_cModel_flag_apCorr
     "@id": "#Object.g_cModel_flag_apCorr"
     datatype: boolean
     description: Set if unable to aperture correct modelfit_CModel. Forced on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2146
   - name: g_calibFlux
     "@id": "#Object.g_calibFlux"
     datatype: double
     description: Flux within 12.0-pixel aperture. Measured on g-band.
     fits:tunit: nJy
+    tap:principal: 0
+    tap:column_index: 2147
   - name: g_calibFluxErr
     "@id": "#Object.g_calibFluxErr"
     datatype: double
     description: Flux uncertainty within 12.0-pixel aperture. Measured on g-band.
     fits:tunit: nJy
+    tap:principal: 0
```

Comment:
This description is accurate as of DP0.2, but would not be accurate today; we need to make sure we don't blindly copy it over.

Reply:
What about it would not be correct today?

Reply:
We've switched from a simple circular aperture to a compensated top-hat filter.  This effectively does a local background subtraction, making the fluxes that go into the photometric calibration less dependent on having a good background subtraction done first.